DuckDuckGo/Common/Localizables/UserText.swift

```diff
@@ -1300,53 +1300,45 @@ struct UserText {
+    // Key: "home.page.promotion.freemium.dbp.post.scan.engagement.result.single.match.text"
+    // Comment: "Text for the Freemium DBP Home Page Post Scan Engagement Promotion When Only One Record is Found"
+    static let homePagePromotionFreemiumDBPPostScanEngagementResultSingleMatchText = "Your free personal info scan found 1 record about you on 1 site."
 
-    // Key: "home.page.promotion.freemium.dbp.post.scan.engagement.result.single.match.subtitle"
-    // Comment: "Subtitle for the Freemium DBP Home Page Post Scan Engagement Promotion When Only One Record is Found"
-    static let homePagePromotionFreemiumDBPPostScanEngagementResultSingleMatchSubtitle = "1 record has been found on 1 site"
-
-    /// Generates a subtitle for the Freemium DBP Home Page Post Scan Engagement Promotion when records are found on a single broker site.
-    /// Key: "home.page.promotion.freemium.dbp.post.scan.engagement.result.single.broker.subtitle"
+    /// Generates Text for the Freemium DBP Home Page Post Scan Engagement Promotion when records are found on a single broker site.
+    /// Key: "home.page.promotion.freemium.dbp.post.scan.engagement.result.single.broker.text"
     ///
     /// - Parameter resultCount: The number of records found.
     /// - Returns: A formatted string indicating the number of records found on 1 site.
-    static func homePagePromotionFreemiumDBPPostScanEngagementResultSingleBrokerSubtitle(resultCount: Int) -> String {
-        String(format: "%d records have been found on 1 site", resultCount)
+    static func homePagePromotionFreemiumDBPPostScanEngagementResultSingleBrokerText(resultCount: Int) -> String {
+        String(format: "Your free personal info scan found %d records about you on 1 site.", resultCount)
     }
 
-    /// Generates a subtitle for the Freemium DBP Home Page Post Scan Engagement Promotion when records are found on multiple broker sites.
-    /// Key: "home.page.promotion.freemium.dbp.post.scan.engagement.result.plural.subtitle"
+    /// Generates Text for the Freemium DBP Home Page Post Scan Engagement Promotion when records are found on multiple broker sites.
+    /// Key: "home.page.promotion.freemium.dbp.post.scan.engagement.result.plural.text"
     ///
     /// - Parameters:
     ///   - resultCount: The number of records found.
     ///   - brokerCount: The number of broker sites where records were found.
     /// - Returns: A formatted string indicating the number of records found on multiple sites.
-    static func homePagePromotionFreemiumDBPPostScanEngagementResultPluralSubtitle(resultCount: Int, brokerCount: Int) -> String {
-        String(format: "%d records have been found on %d sites", resultCount, brokerCount)
+    static func homePagePromotionFreemiumDBPPostScanEngagementResultPluralText(resultCount: Int, brokerCount: Int) -> String {
+        String(format: "Your free personal info scan found %d records about you on %d sites.", resultCount, brokerCount)
```

Comment:
I think the word `different` was missed here 
![image](https://github.com/user-attachments/assets/c352cb75-8bbe-43e3-bb82-18a435fd3c20)